packages/integration-tests/projects/suite-web/tests/suite/database-migration.test.ts

```diff
@@ -1,46 +1,139 @@
+                cy.task('pressYes');
+                cy.getTestElement('@discovery/loader').should('be.visible');
+                cy.getTestElement('@discovery/loader').should('not.exist');
 
-                // add some passphrase wallet
-                // remember wallets
-                // change settings (language, currency, theme)
-                // backends, custom urls
-                // nice to have: labeling
+                // check and store address of first btc tx
+                cy.getTestElement('@suite/menu/wallet-index').click();
+                cy.get('[data-test^="@metadata/outputLabel"]')
+                    .first()
+                    .should('be.visible')
+                    .invoke('attr', 'data-test')
+                    .as('firstTxLabel');
+
+                // remembering and labeling the wallet
+                cy.getTestElement('@menu/switch-device').click();
+                cy.getTestElement('@modal').should('be.visible');
+                cy.contains(hiddenWalletSelector, 'Hidden wallet #1')
+                    .should('be.visible')
+                    .find('[data-test*="add-label"]')
+                    .click({
+                        force: true,
+                    });
+                cy.passThroughInitMetadata(provider);
+                cy.getTestElement('@metadata/input').type(`${testData.walletLabel}{enter}`);
+                cy.contains(hiddenWalletSelector, testData.walletLabel)
+                    .find('[data-test*="toggle-remember-switch"]')
+                    .click()
+                    .find('input')
+                    .should('be.checked');
             }
 
             // KEEP_DB env is truthy.
             // This block will execute only in every other run expect for the first one
             // It means that migration took place and we should see the same data
             else {
                 // asset data was loaded correctly from db
-                cy.getTestElement('@dashboard/graph', { timeout: 30000 }).should('exist');
+                cy.getTestElement('@dashboard/graph', { timeout: 30000 }).should('be.visible');
+                cy.getTestElement('@menu/switch-device').click();
+                cy.contains(hiddenWalletSelector, testData.walletLabel)
+                    .find('input')
+                    .should('be.checked');
+                cy.contains(hiddenWalletSelector, testData.walletLabel).click();
+                cy.getTestElement('@suite/menu/wallet-index').click();
+                cy.get('[data-test^="@metadata/outputLabel"]').first().should('be.visible');
+
+                // check and store address of first btc tx
+                cy.getTestElement('@suite/menu/wallet-index').click();
+                cy.get('[data-test^="@metadata/outputLabel"]')
+                    .first()
+                    .should('be.visible')
+                    .invoke('attr', 'data-test')
+                    .should('eq', this.firstTxLabel);
```

Comment:
+ add something that works with keys. for example show address on device. it should not trigger passphrase dialog